proj/bulgarian.md

```diff
@@ -0,0 +1,278 @@
+# Outdoor Relay Control
+
+<a href="https://www.tindie.com/products/arduino-core/outdoor-relay-control/"><img src="https://d2ss6ovg47m0r5.cloudfront.net/badges/tindie-larges.png" alt="Ad" width="200" height="104"></a>
+
+## Проблеми
+
+`#4` Добавете LCD дисплей
+
+`#5` Добавяне на RTC модул
+
+`#6` Завършете секцията с цели числа
+
+## Компоненти и консумативи
+
+* `1` Arduino Uno R3
+* `1` Mакет 800 точки
+* `1` Mакет 400 точки
+* `?` Женски кабели
+* `?` Мъжки кабели
+* `1` Температурен сензор TMP36
+* `1` ЛСД дисплей
+
+## Инструменти и машини
+
+* `1` 3D принтер
+
+## Линкове
+
+[Програмен код](https://github.com/arduino-core/Outdoor-Relay-Control)
+
+[Документация](https://hackaday.io/project/189172-outdoor-relay-control)
+
+[Дизайнерски файлове](https://wokwi.com/projects/353834225920033793)
+
+[Магазин](https://www.tindie.com/products/arduino-core/outdoor-relay-control/)
+
+## Описание на проекта
+
+[Английски](/proj/english.md)
+
+[Български](/proj/bulgarian.md)
+
+[Немски](/proj/german.md)
+
+### Цели числа
+
+Цяло число е положително или отрицателно число, включително `0`.
+
+```cpp
+int year = 2023;
+```
+
+#### Примери
+
+```cpp
+int zero = 0;
+```
+```cpp
+int positive = 2;
+```
+```cpp
+int negative = -4;
+```
+
+#### Използвани цели числа
+
+В следващия списък ще видите всички използвани цели числа в проекта. Дефиницията на целите числа е [тук](#Дефинирани-цели-числа).
+
+```cpp
+int potentiometer_value1;
+```
+```cpp
+int potentiometer_value2;
+```
+```cpp
+int potentiometer_value3;
+```
+```cpp
+int potentiometer_value4;
+```
+```cpp
+int potentiometer_analogValue1;
+```
+```cpp
+int potentiometer_analogValue2;
+```
+```cpp
+int potentiometer_analogValue3;
+```
+```cpp
+int potentiometer_analogValue4;
+```
+```cpp
+int potentiometer_temperature_min;
+```
+```cpp
+int potentiometer_temperature_max;
+```
+```cpp
+int temperature_value;
+```
+```cpp
+int temperature;
+```
+```cpp
+int potentiometer_clock_min;
+```
+```cpp
+int potentiometer_clock_max;
+```
+```cpp
+int clock_time;
+```
+
+#### Дефинирани цели числа
+
+Следният списък съдържа дефиниции на използвани цели числа в този проект.
+
+```cpp
+int potentiometer_value1 = 0;
+```
+```cpp
+int potentiometer_value2 = 0;
+```
+```cpp
+int potentiometer_value3 = 0;
+```
+```cpp
+int potentiometer_value4 = 0;
+```
+```cpp
+int potentiometer_analogValue1 = 0;
+```
+```cpp
+int potentiometer_analogValue2 = 0;
+```
+```cpp
+int potentiometer_analogValue3 = 0;
+```
+```cpp
+int potentiometer_analogValue4 = 0;
+```
+```cpp
+int potentiometer_temperature_min = 0;
+```
+```cpp
+int potentiometer_temperature_max = 0;
+```
+```cpp
+int temperature_value = 0;
+```
+```cpp
+int temperature = 0;
+```
+```cpp
+int potentiometer_clock_min = 0;
+```
+```cpp
+int potentiometer_clock_max = 0;
+```
+```cpp
+int clock_time = 0;
+```
+
+### `void`
```

Comment:
Since void is a method, I decided not to translate it.

Reply:
@MitkoVtori Good. Please don't translate any code or headings as code formatted.

Reply:
Ok. Lets merge